clang/docs/checkedc/3C/INSTALL.md

```diff
@@ -0,0 +1,54 @@
+# Building 3C
+
+This document describes our current setup and build process for 3C.  You may find helpful information about topics not addressed here in the [general Checked C build documentation](../Setup-and-Build.md).  Some items here are potentially applicable to Checked C as a whole, and we hope to eventually harmonize them with the Checked C documentation as appropriate.
+
+## Supported environments
+
+We work primarily on Linux.  Checked C supports Windows, but we do not test 3C regularly on Windows.  Some of us use Mac OS X, which Checked C does not officially support; we have gotten 3C to mostly work with some additional steps (see below), but there are some known problems and [Microsoft is still considering whether to work with us to address them](https://github.com/microsoft/checkedc/issues/424#issuecomment-720648442).
+
+## Basics
+
+**(TODO: Test from start to finish!)**
+
+Assuming you have already cloned the `checkedc-clang` repository, run the following (from the root of the working tree) for a basic build:
+
+```
+# See https://github.com/correctcomputation/checkedc-clang/issues/296 about this.
+(llvm/projects/checkedc-wrapper && git clone https://github.com/microsoft/checkedc)
+
+mkdir build && cd build
+cmake ../llvm -G Ninja -DLLVM_ENABLE_PROJECTS=clang
+ninja TARGET
+```
+
```

Comment:
Please mention cloneing the checkedc repo with the header files to the wrapper directory. I've tested it to work and it simplifies directions greatly

Reply:
Tha's what the `(llvm/projects/checkedc-wrapper && git clone https://github.com/microsoft/checkedc)` line above does.

Reply:
OK, but then drop the reference to the issue. I don't see the benefit of that. If the above works, then that's great, let's use it.

Reply:
As you prefer.  My standard practice is that a hack should include information of some kind about why it is needed.

If you could remember to "unresolve" review threads when appropriate, that would save me some work of looking through carefully to make sure I didn't miss anything.